Backpack/Theme/Classes/BPKThemeContainerController.m

```diff
@@ -34,12 +34,23 @@ - (void)dealloc {
 
 - (instancetype)initWithThemeDefinition:(id<BPKThemeDefinition>)themeDefinition
                      rootViewController:(nonnull UIViewController *)rootViewController {
+    self = [self initWithThemeContainer:[BPKTheme containerFor:themeDefinition] rootViewController:rootViewController];
+
+    if (self) {
+        _themeDefinition = themeDefinition;
+    }
+
+    return self;
+}
+
+- (instancetype)initWithThemeContainer:(UIView *)themeContainer
+                    rootViewController:(nonnull UIViewController *)rootViewController {
     self = [super initWithNibName:nil bundle:nil];
 
     if (self) {
         _themeActive = YES;
-        _themeDefinition = themeDefinition;
-        _themeContainer = [BPKTheme containerFor:_themeDefinition];
+        _themeDefinition = nil;
```

Comment:
This is now lying in the public interface because `themeDefintion` is `nulalble`. Maybe it's better to make super class that knows nothing about `BPKThemeDefintion` instead?